src/main/java/org/openapitools/client/model/ExecuteFilterParameter11.java

```diff
@@ -13,16 +13,14 @@
 
 package org.openapitools.client.model;
 
-import java.util.Objects;
+import java.util.*;
```

Comment:
I would only bring in what u need - setup ur intellij code style to not do this. These are massive packages in some cases